oeps/architectural-decisions/oep-0030-arch-pii-markup-and-auditing.rst

```diff
@@ -80,11 +81,11 @@ Information that is considered PII in Open edX includes, but is not limited to:
 - Forum posts
 - Notes service entries
 
-Each case where we store data relating to a learner should be considered individually as to whether that information alone, or combined with other stored information, can be used to identify a learner or violate their privacy. In some cases PII must be kept for legal reasons that supersede the need to forget. If there is any doubt about what constitutes PII or whether a specific piece of information can be safely forgotten, seek legal assistance.
+Each case where we store data relating to a learner should be considered individually as to whether that information alone, or combined with other stored information, can be used to identify a learner or violate their privacy. In some cases PII must be kept for legal reasons that supersede the need to forget. If there is any doubt about what constitutes PII or whether a specific piece of information can be safely forgotten, seek assistance from Axim.
```

Comment:
Take it or leave it but possibly link to our ticket system?

```suggestion
Each case where we store data relating to a learner should be considered individually as to whether that information alone, or combined with other stored information, can be used to identify a learner or violate their privacy. In some cases PII must be kept for legal reasons that supersede the need to forget. If there is any doubt about what constitutes PII or whether a specific piece of information can be safely forgotten, seek `assistance from Axim`_.
```

`.. _assistance from Axim: https://github.com/openedx/axim-engineering/issues/new/choose

Reply:
Good call